src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -208,59 +210,61 @@
 
         <!-- Manage Competition Modal -->
         <div class="ui manage-competition modal">
-            <a href="{URLS.COMPETITION_EDIT(competition.id)}" class="ui blue button">Edit competition</a>
-            <button class="ui button published icon { grey: !competition.published, green: competition.published }"
-                    onclick="{ toggle_competition_publish }">
-                <i class="icon file"></i> {competition.published ? "Published" : "Draft"}
-            </button>
-            <button class="ui yellow button icon" onclick="{create_dump}">
-                <i class="download icon"></i> Create Competition Dump
-            </button>
-            <button class="ui teal icon button" onclick="{update_files}">
-                <i class="sync alternate icon"></i>
-                Refresh Table
-            </button>
-            <table class="ui table">
-                <thead>
-                <tr>
-                    <th>Files</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr show="{files.bundle}">
-                    <td class="selectable">
-                        <a href="{files.bundle ? files.bundle.url : ''}">
-                            <i class="file archive outline icon"></i>
-                            Bundle: {files.bundle ? files.bundle.name : ''}
-                        </a>
-                    </td>
-                </tr>
-                <tr each="{file in files.dumps}" show="{files.dumps}">
-                    <td class="selectable">
-                        <a href="{file.url}">
-                            <i class="file archive outline icon"></i>
-                            Dump: {file.name}
-                        </a>
-                    </td>
-                </tr>
-                <tr>
-                    <td show="{!files.dumps && !files.bundle}">
-                        <em>No Files Yet</em>
-                    </td>
-                </tr>
-                </tbody>
-            </table>
+            <div class="content">
```

Comment:
This whole "addition" is just wrapping the modal content in a `div.content` so semantic would pad the modal correctly

Reply:
ty for explanation :)